src/ape/managers/accounts.py

```diff
@@ -97,6 +104,20 @@ def __getitem_str(self, account_str: str):
     def __contains__(self, address: AddressType) -> bool:  # type: ignore
         return any(address in container for container in self.containers.values())
 
+    def generate_test_account(self) -> TestAccount:
```

Comment:
Here are my thoughts here:

1. Make the return type a `TestAccountAPI` (more generic)
2. Add abstract method `generate_account` to the `TestAccountContainerAPI` class
3. Move bulk of this logic to the `TestAccountContainer` class in `ape_test`
4. Change this implementation to:

```python
container = self.containers["test"]
return container.generate_account()
```

It is not perfect and does not 100% mitigate the concerns I have here, but it is at least closer in the right direction.

Reply:
Could even make this a bit more flexible via 
```suggestion
    def generate_test_account(self, container_name: str = "test") -> TestAccountAPI:
        return self.containers[container_name].generate_account()
```

with the above-mentioned changes to `TestAccountContainerAPI`